internal/runner/detector.go

```diff
@@ -13,7 +13,9 @@ type RunnerConfig struct {
 	TestFilePattern        string
 	TestFileExcludePattern string
 	RetryTestCommand       string
-	ResultPath             string
+	// ResultPath is used internally so bktec can read result from Test Runner.
+	// User typically don't need to worry about setting this except in Rspec.
```

Comment:
```suggestion
	// User typically don't need to worry about setting this except in Rspec, and Playwright.
```
Playwright result needs to be configured in `config.js`, and bktec doesn't have a way to know that, so its' mandatory for Playwright.

Reply:
Thanks! fixed 👍🏿 . I wonder if we can just automatically mutate `config.js` on runtime 🤔